api/src/main/java/org/openmrs/module/patientflags/api/impl/FlagServiceImpl.java

```diff
@@ -287,7 +336,19 @@ public void purgeTag(Integer tagId) {
 	public Priority getPriority(Integer priorityId) {
 		return dao.getPriority(priorityId);
 	}
-	
+	/**
+	 * @see org.openmrs.module.patientflags.api.FlagService#getPriorityByUuid(String)
+	 */
+	public Priority getPriorityByUuid(String uuid) {
+		return dao.getPriorityByUuid(uuid);
+	}

```

Comment:
You need to have at least a line between methods.

Reply:
Done